src/gnosisSafe/transactions/notifiedTransactions.js

```diff
@@ -0,0 +1,11 @@
+export const TX_NOTIFICATION_TYPES = {
```

Comment:
I think we can remove this one also `TX_NOTIFICATION_TYPES` file, as we are not using anywhere currently.